hal/targets/hal/TARGET_STM/TARGET_STM32F0/lp_ticker.c

```diff
@@ -0,0 +1,49 @@
+#include "device.h"
+
+#if DEVICE_LOWPOWERTIMER
+
+#include "ticker_api.h"
+#include "lp_ticker_api.h"
+#include "rtc_api.h"
+#include "rtc_api_hal.h"
+
+static uint8_t lp_ticker_inited = 0;
+static uint32_t m_timestamp = 0;
+static uint32_t m_offset = 0;
+
+void lp_ticker_init() {
+    /*
+     * At this stage we don't have the users requested time.
+     * This is only to cover the API implementation.
+     */
+}
+
+uint32_t lp_ticker_read() {
+    /*
+     * Becuase our RTC has a periodic wake-up timer we just want to simply
+     * return the desired timestamp value to configure RTC exactly with the value
+     * provided by the user and to fulfill the mBed ticker architecture.
+     */
+    return m_timestamp;
```

Comment:
https://github.com/mbedmicro/mbed/blob/master/hal/hal/lp_ticker_api.h#L56

This should return current counter. how does it work for a use case like:

```
start = lp_ticker_read();
while (start + 10000 < lp_ticker_read()) { 
  sleep(); 
}
```